### PR TITLE
Revamp READMEs

### PR DESCRIPTION
**Pull request by melanieseltzer**
Rendered:

Root:
https://github.com/ngrok-oss/mantle/tree/readme-revamp#readme

Package readme (will show on npm):
https://github.com/ngrok-oss/mantle/tree/readme-revamp/packages/mantle#readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,91 +1,23 @@
-# Mantle
+<p align="center">
+	<picture>
+		<source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/ngrok-oss/mantle/refs/heads/readme-revamp/.github/mantle-dark.svg">
```
Comment (melanieseltzer) [inline README.md:3]: Once approved, I will change all these URLs to `https://raw.githubusercontent.com/ngrok-oss/mantle/HEAD/` and not reference this branch.

**Review by aaronshekey — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,91 +1,23 @@
...
+  Mantle Design System
+</h1>
 
 [Mantle](https://mantle.ngrok.com) is [ngrok](https://ngrok.com/)’s UI library and design system. It’s inspired by [shadcn/ui](https://ui.shadcn.com) and uses [Radix](https://www.radix-ui.com) with [Tailwind](https://tailwindcss.com/) for styling. It’s documented using [Remix](https://remix.run/docs).
 
-## Get Started
+## Getting Started
 
-To get started, install `@ngrok/mantle` with your preferred package manager:
+Check out our [install instructions](https://github.com/ngrok-oss/mantle/tree/main/packages/mantle#readme).
```
Comment (cody-dot-js) [inline README.md:16]: Can't you do relative path references here?

Comment (melanieseltzer) [reply README.md:16]: _Devs don't read_ https://github.com/ngrok-oss/mantle/pull/345#discussion_r1826020202 😏

Comment (cody-dot-js) [reply README.md:16]: No, not that. The reference to the sub mantle package readme

Comment (melanieseltzer) [reply README.md:16]: Oh wait it is I who cannot read, you're not talking about the image URLs. Yeah I'll fix this!

:homer-in-bushes-emoji:

**Review by cody-dot-js — APPROVED** [review]
🦐

**Pull request merged**